content/core-concepts/metrics.md

```diff
@@ -69,21 +69,26 @@ export their own views and claim the view names by registering them.
 #### Sampling
 
 Stats are NOT sampled to be able to represent uncommon
-cases. For example, a [99th percentile latency issue](https://www.youtube.com/watch?v=lJ8ydIuPFeU)
-is rare. Combined with a low sampling rate,
-it might be hard to capture it. This is why stats are not sampled.
+cases hence, stats are ALWAYS recorded unless [\[1\] dropped](#[1]-dropped).
 
-On the other hand, exporting every individual measurement would
-be very expensive in terms of network traffic. This is why stats
-collection aggregates data in the process and exports only the
-aggregated data.
+The reasoning behind this is that outlier statistics such as "99th latency event"
+occur rarely but also occur unpredictably. If our recording system has a low sampling,
+we are most likely to miss recording such events. Hence this is the reason
+why stats are NOT sampled but are ALWAYS recorded.
```

Comment:
Maybe also mention that since stats are already aggregated data, they should be pretty light-weight (i.e n `ViewData` correspond to n `View`s). I think that's another reason stats are not sampled.

Reply:
Would you mind providing a follow-up PR explaining this? I don't see a simple sentence to articulate this.